### DISCUSSION TITLE
Problem to save a Virtual Dataset!

### DISCUSSION THREAD

**Discussion by Manjavachi** [apache/superset#25984] [Q&A / Help]
Ee're testing superset, at my company, but every time I tried to save a virtual dataset from an SQL Querry I receive this message error:

'JSON.parse: unexpected character at line 4 column of the JSON data'. 

Anyone can point me where can I start looking to solve this problem?

**Comment by luis-doudeau** [comment]
The same problem, please help

**Comment by hiwaveSupport** [comment]
Hard to pin point with information on hand, but looks like it is breaking the json parser which means your data has a special character that makes the json invalid? Have you tried creating it with a very simple dataset e.g. name, city two text fields which you have in a sql db that you can convert to virtual dataset?

**Comment by klugjohannes** [comment]
I'm running into the same problem on Superset version 2.1.3, hosted through the non-dev docker compose stack.

When I open the developer console it seems to me, superset is trying to redirect to the login page.

![grafik](https://github.com/apache/superset/assets/888995/8117975e-e0d7-4bcd-b202-cd33408e9411)

I first thought it might be due to CSRF issues, as I'm tunneling the application to localhost:8080 through ssh, but even accessing service directly and setting the SUPERSET_SESSION_COOKIE_SAMESITE=None environment variable leads to no improvements.
